src/main/java/workers/ProfessorRatingScraper.java

```diff
@@ -0,0 +1,54 @@
+package workers;
+
+import java.io.IOException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.*;
+import java.util.HashMap;
+import java.util.Map;
+
+class ProfessorRatingScraper {
+    private static Map<String, Double> previousScores = new HashMap<String, Double>();
+
+    public static double getRating(String firstName, String lastName) {
+        if (previousScores.containsKey(firstName + lastName)) {
+            return previousScores.get(firstName + lastName);
+        }
+        String html;
+        try {
+            html = Jsoup.connect(
+                    "https://www.ratemyprofessors.com/search/teachers?query="
+                            + firstName
+                            + " "
+                            + lastName
+                            + "&sid=U2Nob29sLTE0ODQ=")
+                    .get()
+                    .html();
+        } catch (IOException e) {
+            return 2.5;
```

Comment:
This is a big no no, throw the error. I see that we need a return but you could refactor it to just throw the error instead w/out having to return 2.5. Up to you, at the very least throw the error.